e2e/pages/session-search.po.ts

```diff
@@ -75,6 +76,36 @@ export class SessionSearchPage {
         await row.element(by.cssContainingText('.clickable', 'Edit')).click()
     }
 
+    async checkIfHasNote(expectedNoteId, expectedNoteValue, ...values: string[]): Promise<boolean> {
```

Comment:
this method is a bit too big. Can you split the responsibility to separate methods?